ts/features/fci/components/ErrorComponent.tsx

```diff
@@ -64,63 +69,81 @@ const ErrorComponent = (props: Props) => {
     }
   };
 
-  const retryButtonProps = {
+  const retryButtonProps: ButtonSolidProps = {
     testID: "FciRetryButtonTestID",
-    block: true,
-    primary: true,
     onPress: props.onPress,
-    title: I18n.t("features.fci.errors.buttons.retry")
+    fullWidth: true,
+    label: I18n.t("features.fci.errors.buttons.retry"),
+    accessibilityLabel: I18n.t("features.fci.errors.buttons.retry")
   };
 
-  const closeButtonProps = {
+  const closeButtonProps: ButtonSolidProps = {
     testID: "FciCloseButtonTestID",
-    bordered: true,
-    block: true,
     onPress: props.onPress,
-    title: I18n.t("features.fci.errors.buttons.close")
+    fullWidth: true,
+    label: I18n.t("features.fci.errors.buttons.close"),
+    accessibilityLabel: I18n.t("features.fci.errors.buttons.close")
   };
 
-  const assistanceButtonProps = {
+  const assistanceButtonProps: ButtonSolidProps = {
     testID: "FciAssistanceButtonTestID",
-    bordered: true,
-    primary: false,
-    block: true,
+    fullWidth: true,
     onPress: handleAskAssistance,
-    title: I18n.t("features.fci.errors.buttons.assistance")
+    label: I18n.t("features.fci.errors.buttons.assistance"),
+    accessibilityLabel: I18n.t("features.fci.errors.buttons.assistance")
   };
 
+  /**
+   * Render the footer buttons as vertical stacked buttons
+   * @returns {React.ReactElement}
+   */
   const footerButtons = () => {
     if (props.retry && props.assistance) {
-      return [retryButtonProps, assistanceButtonProps];
+      return (
+        <>
+          <ButtonSolid {...retryButtonProps} />
+          <VSpacer size={8} />
+          <ButtonOutline {...assistanceButtonProps} />
+        </>
+      );
     }
     if (props.retry) {
-      return [retryButtonProps, closeButtonProps];
+      return (
+        <>
+          <ButtonSolid {...retryButtonProps} />
+          <VSpacer size={8} />
+          <ButtonOutline {...closeButtonProps} />
+        </>
+      );
     }
     if (props.assistance) {
-      return [
-        {
-          ...closeButtonProps,
-          bordered: false,
-          title: I18n.t("features.fci.errors.buttons.back")
-        },
-        assistanceButtonProps
-      ];
+      return (
+        <>
+          <ButtonSolid
+            {...{
+              ...closeButtonProps,
+              label: I18n.t("features.fci.errors.buttons.back"),
+              accessibilityLabel: I18n.t("features.fci.errors.buttons.back")
+            }}
+          />
+          <VSpacer size={8} />
+          <ButtonOutline {...assistanceButtonProps} />
+        </>
+      );
     }
-    return [closeButtonProps];
+    return <ButtonOutline {...closeButtonProps} />;
```

Comment:
Is this supposed to be a `ButtonSolid`? 
| Solid | Outline |
|--------|--------|
| <img src="https://github.com/pagopa/io-app/assets/12371664/09a5ded4-c3cb-4720-a09e-6114d5a83b69" height="400"> | <img src="https://github.com/pagopa/io-app/assets/12371664/04e751c7-82ff-4933-a6d6-5cade20e5533" height="400"> |

Reply:
the actual figma (v1.0) seems to use outline